### PR TITLE
Test against `anyio` master branch

### PR DESCRIPTION
**Pull request by tomchristie**
We'd like to pre-emptively bump our requirements for any upcoming anyio 4.0 release.
See https://github.com/encode/httpcore/pull/596

Issuing this PR just to run our CI against the latest master version.
Not intending to merge this.

### CODE REVIEW

**Comment by tomchristie** [comment]
Okay, I've obviously got the syntax wrong here. 🤷‍♂️

**Comment by agronholm** [comment]
The correct syntax is `anyio @ git+https://github.com/agronholm/anyio.git`.

File: requirements.txt [hunk 1]
```diff
@@ -16,7 +16,7 @@ twine
 wheel
 
 # Tests & Linting
-anyio==3.6.1
+anyio==git+https://github.com/agronholm/anyio.git
```
Comment (Kludex) [inline requirements.txt:19]:
```suggestion
anyio @ git+https://github.com/agronholm/anyio.git
```

**Comment by Kludex** [comment]
passes :+1:

**Pull request closed without merging**